packages/opencensus-instrumentation-grpc/src/grpc-stats/stats-common.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {globalStats} from '@opencensus/core';
+import {GRPC_BASIC_CLIENT_VIEWS} from './client-metrics';
+import {GRPC_BASIC_SERVER_VIEWS} from './server-metrics';
+
+// Common histogram bucket boundaries for bytes received/sets Views.
```

Comment:
Nit: could you make this a JSDoc comment `/** */` and same for `DEFAULT_MILLI_SECONDS_DISTRIBUTION` and `DEFAULT_MESSAGE_COUNT_DISTRIBUTION` below too?